File: .claude/worktrees/gifted-hugle-ba34b1/functions/src/config/app_urls.ts
/**
 * Single source of truth for all whenote.app URLs and contact emails
 * used across Cloud Functions.
 *
 * Every deep-link, legal page URL, or sender email that references the
 * `whenote.app` domain MUST come from this file so that a domain change
 * only requires editing one place.
 */

// ---------------------------------------------------------------------------
// Base
// ---------------------------------------------------------------------------
export const BASE_URL = "https://whenote.app";

// ---------------------------------------------------------------------------
// Deep links
// ---------------------------------------------------------------------------
export const letterUrl = (letterId: string): string =>
  `${BASE_URL}/letter/${letterId}`;

export const openUrl = (token: string): string =>
  `${BASE_URL}/open/${token}`;

// ---------------------------------------------------------------------------
// Legal / policy pages
// ---------------------------------------------------------------------------
export const PRIVACY_URL = `${BASE_URL}/privacy.html`;
export const TERMS_URL = `${BASE_URL}/terms.html`;

// ---------------------------------------------------------------------------
// Email defaults
// ---------------------------------------------------------------------------
export const DEFAULT_FROM_EMAIL =
  process.env.SENDGRID_FROM_EMAIL || "noreply@whenote.app";

File: .claude/worktrees/xenodochial-albattani-c62292/lib/core/constants/app_urls.dart
/// Single source of truth for all whenote.app URLs and contact emails.
///
/// Every deep-link, legal page URL, or contact email that references the
/// `whenote.app` domain MUST come from this file so that a domain change
/// only requires editing one place.
class AppUrls {
  AppUrls._();

  // ---------------------------------------------------------------------------
  // Base
  // ---------------------------------------------------------------------------
  static const String baseUrl = 'https://whenote.app';

  // ---------------------------------------------------------------------------
  // Deep links
  // ---------------------------------------------------------------------------
  static String letterUrl(String docId) => '$baseUrl/letter/$docId';
  static String capsuleUrl(String docId) => '$baseUrl/capsule/$docId';
  static String openUrl(String token) => '$baseUrl/open/$token';

  // ---------------------------------------------------------------------------
  // Legal / policy pages
  // ---------------------------------------------------------------------------
  static const String privacyUrl = '$baseUrl/privacy.html';
  static const String termsUrl = '$baseUrl/terms.html';

  // ---------------------------------------------------------------------------
  // Contact emails
  // ---------------------------------------------------------------------------
  static const String supportEmail = 'suporte@whenote.app';
  static const String privacyEmail = 'privacidade@whenote.app';
  static const String legalEmail = 'juridico@whenote.app';
  static const String dpoEmail = 'dpo@whenote.app';
}
